source/extensions/upstreams/http/BUILD

```diff
@@ -12,6 +12,7 @@ envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
+    category = "SOMECAT",
```

Comment:
i see

```
$ grep upstream catlist.txt 
envoy.filters.upstream_network
envoy.transport_sockets.upstream
envoy.upstream_options
envoy.upstreams
```

and im guessing `envoy.upstreams`

Reply:
Yes